modules/dev-tools/src/configuration/get-esbuild-config.ts

```diff
@@ -67,6 +67,25 @@ function umdWrapper(libName: string | undefined) {
   };
 }
 
+/**
+ * ESBuild plugin to inline important ESM-only dependencies, for CJS compatibility.
+ * Reference: https://github.com/evanw/esbuild/issues/3442
+ */
+const inlineESMOnly = (): Plugin => {
+  const packageRoot = process.cwd();
+
+  return {
+    name: 'inline-esm-only',
+    setup(build) {
+      // TODO: Detect ESM-only from package.json, instead of hard-coding package names?
```

Comment:
If this is difficult to do, can we make it a ocularrc config instead of hard-coding it here?